pkg/build/builder/docker.go

```diff
@@ -12,7 +12,6 @@ import (
 	dockercmd "github.com/docker/docker/builder/command"
 	"github.com/docker/docker/builder/parser"
 	docker "github.com/fsouza/go-dockerclient"
-	"github.com/golang/glog"
```

Comment:
i guess i can appreciate keeping the name the same so people writing code don't have to think about it, but i also dislike the idea that people are using it w/o realizing it's different. 
[shrug].